pyproject.toml

```
@@ -1,20 +1,130 @@
+[project]
+name = "brainglobe-registration"
+authors = [{name = "Adam Tyson", email= "code@adamltyson.com"}]
```

I don't think I can claim much credit for this repo anymore! For long term support reasons, I wonder if we should start using something like this across all repos?
```suggestion
authors = [{name = "Igor Tatarnikov", email= "hello@brainglobe.info"}]
```

Any thoughts @alessandrofelder?

---

Yea, or even something like
```
authors = [{name = "Brainglobe Developers", email= "hello@brainglobe.info"}]
```
Or even "Brainglobe Core Developers"?

---

Especially on repos where we all contribute significantly... this one is definitely by and large just Igor so far!

---

Yeah I dunno. The single author entry doesn't really tally with how OSS is developed. OTOH, I want to be able to promote individuals as much as possible.

---

although nobody reads this, so maybe BG developers is ok.

---

Made this a discussion for the [next dev meeting](https://brainglobe.zulipchat.com/#narrow/stream/414089-developer-meeting/topic/developer-meeting-07-12-2023/near/404686128)!

---

> OTOH, I want to be able to promote individuals as much as possible. 

Agree - I wonder how other OSS projects do this :thinking: maybe a yearly updated list of who was an official BG dev in the last year?

---

[numpy use the generic "NumPy Developers"](https://github.com/numpy/numpy/blob/c09c89cdf6f139e7fc473cd05b46f78bfbb8b1b8/pyproject.toml#L17)

---

I'm partial to:
```suggestion
authors = [{name = "Brainglobe Developers", email= "hello@brainglobe.info"}]
```

---

Fine by me.